config/manifests/bases/percona-xtradb-cluster-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,205 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+    - mysql
+    - percona
+    - database
+    - pxc
+    - galera
+  links:
+    - name: Percona
+      url: 'https://www.percona.com/'
+    - name: Percona Kubernetes Operators Landing Page
+      url: 'https://www.percona.com/software/percona-kubernetes-operators'
+    - name: Documentation
+      url: 'https://percona.github.io/percona-xtradb-cluster-operator/'
+    - name: Github
+      url: 'https://github.com/percona/percona-xtradb-cluster-operator'
+  maintainers:
+    - name: Percona
+      email: info@percona.com
+  maturity: stable
+  provider:
+    name: Percona
+    url: https://github.com/percona/percona-xtradb-cluster-operator
+  version: 1.12.0
```

Comment:
why version 1.12.0?

Reply:
@dkhachyan please let us know if you're willing to work on this further. otherwise we'll takeover this PR.